packages/react/src/lib/VoiceProvider.tsx

```diff
@@ -49,15 +49,8 @@ export type VoiceContextType = {
   isAudioMuted: boolean;
   isPlaying: boolean;
   messages: (
-    | Hume.empathicVoice.UserMessage
-    | Hume.empathicVoice.AssistantMessage
+    | (Hume.empathicVoice.JsonMessage & { receivedAt: number })
```

Comment:
This should be `receivedAt: Date` right? I guess we also need to bump the `hume` version to 0.8.3 too?